cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -118,6 +118,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
         is_entrance_exam=True
     )
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
+        display_name='Exam Sequential - Subsection'
```

Comment:
I think we'd like to internationalize display_name, so just wrap this with a _('....') If this is a human visible string I think @srpearce might want to see if she can come up with a better name